bluemira/equilibria/find_legs.py

```diff
@@ -34,16 +37,22 @@ class NumNull(Enum):
     """
 
     DN = auto()
+    # Double Null
```

Comment:
Same for the other 3, I meant this:
```suggestion
    """Double Null"""
```

Reply:
There is no 'of course!' emoji. Otherwise it would go here.